pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/LLCSegmentCompletionHandlers.java

```diff
@@ -58,8 +65,10 @@
 @Path("/")
 public class LLCSegmentCompletionHandlers {
 
+  public static final String SEGMENT_TMP_DIR = "segment.tmp";
```

Comment:
private is fine, right? We don't use it outside this file

Reply:
Done.